patches/minecraft/net/minecraft/entity/monster/EntityCreeper.java.patch

```diff
@@ -0,0 +1,50 @@
+--- ../src-base/minecraft/net/minecraft/entity/monster/EntityCreeper.java
++++ ../src-work/minecraft/net/minecraft/entity/monster/EntityCreeper.java
+@@ -1,5 +1,6 @@
+ package net.minecraft.entity.monster;
+ 
++import java.util.ArrayList;
```

Comment:
@mezz I know that this will lead to duplicated events, but according to EntityPlayer (exact same behaviour already implemented) this shouldn't be a problem. Only other solution would be to 1 add a hasEventBeenCalled parameter to EntityLiving#onDeath() or b) make that method return a boolean value.